packages/protocol/thread/bounds.ts

```diff
@@ -1,18 +1,15 @@
 import { ObjectId, Rect } from "@replayio/protocol";
 
 import { DisallowEverythingProxyHandler } from "../utils";
-import { Pause } from "./pause";
 
 // Used by the highlighter when showing the bounding client rect of a node
 // that might not be loaded yet, for the node picker.
 export class NodeBoundsFront {
-  private _pause: Pause;
   private _rects: Rect[];
   nodeId: ObjectId;
   then = undefined;
 
-  constructor(pause: Pause, nodeId: ObjectId, rects: Rect[]) {
-    this._pause = pause;
+  constructor(nodeId: ObjectId, rects: Rect[]) {
```

Comment:
Wouldn't this be more accurately/consistently typed as `PauseId`?

Both happen to be aliases for `string` but it still seems worth changing.

Reply:
This isn't used anymore anyway so I'll just delete the file.